Update CHANGELOG.md for proposals editor improvements (#281)
apollographql/apollo-studio-community: CHANGELOG.md

[comment]: <> "NOTE! Ensure all images are added via the \[label\]\(link\) syntax!"

## 2024-07-25
| What's new | |
| :--------- | :-: |

> | Schema proposals editor loading performance has been optimized so you can start editing and collaborating on schema proposals quicker than ever before. Go to `Propsals > Editor` to try it out. | ![image](https://github.com/user-attachments/assets/78e7886b-3eb7-4a13-b028-209be99f028f)

Reviewer: ```suggestion
| We've improved the schema proposals editor's loading performance so you can start editing and collaborating on schema proposals quicker than ever before. Go to `Proposals > Editor` to try it out. | ![image](https://github.com/user-attachments/assets/78e7886b-3eb7-4a13-b028-209be99f028f)
```
